### PR TITLE
Run tests on PHP 8.0 using symfony/phpunit-bridge and fix PHP 8 notices

### PR DESCRIPTION
**Pull request by alexpott**

### CODE REVIEW

**Comment by alexpott** [comment]
Yay nightly PHP 8.0 build is green... https://travis-ci.org/github/TYPO3/phar-stream-wrapper/builds/740085498

File: .scrutinizer.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+build:
+    environment:
+        php: '7.2'
+    dependencies:
+        before:
+            - pecl install xdebug
+            - echo zend_extension=$(php-config --extension-dir)/xdebug.so >> $(php --ini | grep -F '/php.ini' | cut -d ':' -f 2)
+#            - composer install --ignore-platform-reqs
```
Comment (andypost) [inline .scrutinizer.yml:8]: nitpick, looks this line useless

**Review by andypost — APPROVED** [review]
Looks ready to go, just wondered amount of commits)

**Comment by ohader** [comment]
I took the freedom to squash the changes... great work @alexpott! 👍

**Comment by ohader** [comment]
Congrats, it looks like this is a PHP "change" (regression?). Now notices as thrown in PHP 7.3.24 and 7.4.12 as well... see #66

**Pull request merged**